common/src/main/java/net/william278/huskclaims/claim/ClaimHandler.java

```diff
@@ -143,7 +144,36 @@ default boolean cancelNature(@NotNull OperationWorld world,
 
     // Checks if the outcome of an operation is being ignored by its involved user
     private boolean isOperationIgnored(@NotNull Operation operation) {
-        return operation.getUser().map(user -> isIgnoringClaims((OnlineUser) user)).orElse(false);
+        return operation.getUser().map(user -> {
+            final OnlineUser onlineUser = (OnlineUser) user;
+            if (!isIgnoringClaims(onlineUser)) {
+                return false;
+            }
+            
+            // Check if user has permission to bypass this specific operation type
+            return hasIgnoreClaimsPermission(onlineUser, operation.getType());
+        }).orElse(false);
+    }
+
+    // Checks if a user has permission to bypass a specific operation type when ignoring claims
+    private boolean hasIgnoreClaimsPermission(@NotNull OnlineUser user, @NotNull OperationType operationType) {
+        // Check for wildcard permission first
+        if (getPlugin().canUseCommand(IgnoreClaimsCommand.class, user, "operations", "*")) {
+            return true;
+        }
+        
+        // Check for specific operation type permission
+        return getPlugin().canUseCommand(IgnoreClaimsCommand.class, user, "operations", operationType.name());
```

Comment:
i believe we need to manually reformat operationType#name to ensure its in the unprefixed snake case (e.g. this may return CAPS_STUFF or custom_plugin:optype)